frontend/src/layout/panel-layout/ProjectTree/ProjectTree.tsx

```diff
@@ -117,7 +121,15 @@ export function ProjectTree(): JSX.Element {
             ) : null}
             {item.record?.type === 'folder' || item.id?.startsWith('project-folder-empty/') ? (
                 <>
-                    {!item.id?.startsWith('project-folder-empty/') ? <ContextMenuSeparator /> : null}
+                    {!item.id?.startsWith('project-folder-empty/') ? (
+                        <>
+                            {type === 'context' ? (
+                                <ContextMenuSeparator />
+                            ) : type === 'dropdown' ? (
+                                <DropdownMenuSeparator />
+                            ) : null}
+                        </>
+                    ) : null}
```

Comment:
nit: you could just pass down the Separator class as a prop to this same function the way we pass `MenuItem`... or then make `MenuItem` derive its class on the `context` as well.